content/SCALE/GettingStarted/VMandAppConfigSCALE.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Configuring Virtualization and Apps in TrueNAS SCALE"
+weight: 50
+tags:
+- scalevm
+- scaleapps
+- scalenetwork
+- scalevlan
+- scalelagg
+- scaledatashare
+---
+
+{{< toc >}}
+
+Configuring TrueNAS SCALE to work with virtualized features, such as VMs and Apps, is a part of the setup process that when optimized takes advantage of the network storage capabilities that SCALE offers. 
+
+## Before You Begin
+
+This article assumes that you have the latest release version of TrueNAS SCALE installed on your system. The following steps comprise a list of configuration prerequisites that you have completed and are familiar with before beginning VM and App installation.
+
+### Network Prerequisites
+
+The primary network interface is configured as part of the SCALE installation process. Go to **Network > Global Configuration** screen in the TrueNAS web UI to determine that the default gateway, host name, domain and DNS name servers have been configured correctly. See [Console Setup Menu Configuration]({{< relref "ConsoleSetupMenuScale.md" >}}) for more information on network settings.
+
+### Optional: Network VLAN 
+
+You have the option of configuring a virtual LAN (VLAN) to route traffic for your VMs. VLAN benefits include the reduction of broadcast traffic and the ability to group resources in different physical locations into a broadcast domain. VLANs virtually segment a network. Different VLANs can communicate with each other using layer 3 devices. See [Setting Up a Network VLAN]({{< relref "SettingUpVLAN.md" >}}) for more information on creating virtual LANs (VLAN).
+
+### Storage Prerequisites
+
+Storage pool creation is part of the initial process of setting up storage for SCALE. A TrueNAS dataset is a file system within a data storage pool. See [Setting Up Storage]({{< relref "SetUpStorageScale.md" >}}) to review storage pool creation and [Adding and Managing Datasets]({{< relref "DatasetsSCALE.md" >}}) for information on dataset configuration. 
+
+### Set Up Shares
+
+Once the pool and datasets have been created, set up shares to enable data access. Three types of data sharing methods are discussed in 
```

Comment:
Rather than using once, use after because the action occurs after you do something.
```suggestion
After creating the pool and datasets, set up shares to enable data access. Three types of data sharing methods are discussed in 
```